### ISSUE TITLE
Remove validation from Monitor URL field

### ISSUE THREAD

**Issue by Flynsarmy** [bluewave-labs/Checkmate#2605]
**Is your feature request related to a problem? Please describe.**
When running Checkmate inside Docker with a Caddy reverse proxy on a dev machine, from inside containers, they can access other contains via URLs like _https://containername/_. This is an invalid URL and is flagged as such but the URL _https://containername.localhost_ is only accessible from the host machine so Checkmate marks it as down.

**Describe the solution you'd like**
Remove validation on the _URL to monitor_ field of the _Create your monitor_ page.

**Describe alternatives you've considered**
This wouldn't be a problem if the domain was publicly accessible (ie in prod) but that doesn't help me here.

**Additional context**
/caddy/docker-compose.yml:
```yaml
name: caddy-proxy
services:
  caddy:
    image: lucaslorentz/caddy-docker-proxy:alpine
    container_name: caddy
    restart: unless-stopped
    networks:
      - proxy
    ports:
      - 80:80
      - 443:443
    environment:
      - CADDY_INGRESS_NETWORKS=proxy
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock
      - caddy_data:/data
volumes:
  caddy_data:
  caddy_config:

networks:
  proxy:
    name: proxy
    external: true
```

/checkmate/docker-compose.yml:
```yaml
name: checkmate
services:
  server:
    image: ghcr.io/bluewave-labs/checkmate-backend-mono-multiarch:latest
    restart: always
    networks:
      - proxy
      - checkmate
    depends_on:
      - mongodb
    environment:
      - UPTIME_APP_API_BASE_URL=https://checkmate.localhost/api/v1
      - UPTIME_APP_CLIENT_HOST=https://checkmate.localhost
      - DB_CONNECTION_STRING=mongodb://mongodb:27017/uptime_db?replicaSet=rs0
      - REDIS_URL=redis://redis:6379
      - CLIENT_HOST=https://checkmate.localhost
      - JWT_SECRET=my_secret
    labels:
      caddy: checkmate.localhost
      caddy.reverse_proxy: "{{upstreams 52345}}"
    volumes:
    - /var/run/docker.sock:/var/run/docker.sock:ro
  mongodb:
    image: mongo:4.4.18
    restart: always
    command: ["mongod", "--replSet", "rs0", "--bind_ip_all"]
    networks:
      - checkmate
    volumes:
      - ./volumes/mongo/data:/data/db
  mongo-init:
    image: mongo:4.4.18
    depends_on:
      - mongodb
    networks:
      - checkmate
    entrypoint: >
      bash -c "
        echo 'Waiting for MongoDB to be ready...' &&
        until mongo --host mongodb --eval 'db.adminCommand(\"ping\")' > /dev/null 2>&1; do
          sleep 2
        done &&
        echo 'MongoDB is up. Initiating replica set...' &&
        mongo --host mongodb --eval '
          rs.initiate({
            _id: \"rs0\",
            members: [{ _id: 0, host: \"mongodb:27017\" }]
          })
        ' || echo 'Replica set may already be initialized.'
      "
networks:
  proxy:
    name: proxy
    external: true
  checkmate:
```

/some_container/docker-compose.yml:
```yaml
name: dockerpress2
services:
  wordpress:
    image: wordpress:php8.3-fpm
    restart: always
    depends_on:
      - db
    networks:
      - proxy
      - backend
    volumes:
      - ./public:/var/www/html
    labels:
      caddy: wordpress.localhost
      caddy.reverse_proxy: "{{upstreams 443}}"
    environment:
      WORDPRESS_DB_HOST: db
      WORDPRESS_DB_USER: ${DB_USER}
      WORDPRESS_DB_PASSWORD: ${DB_PASS}
      WORDPRESS_DB_NAME: ${DB_NAME}
      WORDPRESS_DEBUG: 1

  db:
    image: mysql:latest
    restart: always
    environment:
      MYSQL_RANDOM_ROOT_PASSWORD: '1'
      MYSQL_USER: ${DB_USER}
      MYSQL_PASSWORD: ${DB_PASS}
      MYSQL_DATABASE: ${DB_NAME}
    volumes:
      - ./volumes/db:/var/lib/mysql
    networks:
      - backend

networks:
  proxy:
    name: proxy
    external: true
  backend:

volumes:
  public:
  db:
```
